src/components/TOCViewer/index.jsx

```diff
@@ -55,6 +55,7 @@ class TreeViewer extends React.Component {
           {
             this.props.node.children ?
               <Button
+                tabindex={this.props.visible ? null : -1}
```

Comment:
think this should be ```tabindex={this.props.visible ? 0 : -1}``

Reply:
@bill-filler yes, `0` and `null` both can be used. both work correctly for our case
But I preferred null over `0` because react simply don't add attributes to DOM with `null` . 
and I think simply skipping tabindex attributes from DOM is better that rendering as `tabindex=0` .

Reply:
what you suggest now ?

Reply:
See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex
I think being explicit with tabindex="0" is preferrable

Reply:
I am totally okay with "0" too. I will update code and merge it after squash.